WordPress/gutenberg
# [RNMobile] Correct Android back button navigation

## Description
That PR is going to fix the issue with Android back button wrong navigation observed during test the other [PR](https://github.com/WordPress/gutenberg/pull/26204#issuecomment-712003420)

## How has this been tested?

1. Open mobile app on Android
2. Add buttons block
3. Open its settings
4. Go to color settings
5. Press custom color
6. Use Android back button to go back
7. Expect to be moved to the color list





## Screenshots 

![andorid_back_button_fix](https://user-images.githubusercontent.com/22746080/98363752-c5dbcc00-202f-11eb-9943-490aa2749f07.gif)

## Types of changes

Removing `setHeight` from array of dependencies which caused the regression







## Checklist:
